crates/hir-def/src/attr.rs

```diff
@@ -853,6 +853,23 @@ impl<'attr> AttrQuery<'attr> {
             .iter()
             .filter(move |attr| attr.path.as_ident().map_or(false, |s| s.to_smol_str() == key))
     }
+
+    pub fn find_string_value_in_tt(self, key: &'attr str) -> Option<&SmolStr> {
```

Comment:
Let's add a comment showing an example for what this specifically does (as the name is not too descriptive, but I also do not have a better one). Something like

/// #[doc(html_root_url = "url")] 
///       ^^^^^^^^^^^^^ key

should suffice